app/equipa/page.tsx

```diff
@@ -2,9 +2,9 @@
 import teamData from "@/data/TeamPageData.json";
 import React, { useState } from "react";
 import DatePicker from "@/components/DatePicker";
-import Teams from "@/components/Team";
+import Team from "@/components/Team";
 
-export default function Team() {
+export default function Equipa() {
```

Comment:
It is a bit weird to have the naming in portuguese 😅
I think the page should be called `team` and so the component.

Reply:
Agreed, i think it is good having the navbar link in portuguese since the whole website is, but the component and filename (therefor, the url) should stay in english.